### ISSUE TITLE
not compatible with Altum-PT?

### ISSUE THREAD

**Issue by rp3team** [micasense/imageprocessing#195]
While working with the Altum-PT image set, I encountered the following error: 
xVignette = float(meta.get_item('XMP:VignettingCenter', 0))
  ==> TypeError: float() argument must be a string or a number, not 'NoneType'.' 

It appears that there is no XMP:VignettingCenter field present in any of the Altum-PT images.

**Comment by HazelSCUT** [comment]
Yes, many codes are not compatible with Altum-PT. I have tried Tutorial 2 and Tutorial 3, it both failed. How could you solve this problem?

**Comment by br-miguez** [comment]
I am having the same problem :/
Did you find any solution?

**Comment by alherca73** [comment]
Has there been any interest in providing solutions for RedEdge-P or Altum-PT?
